I 03-17 21:26:18 controller.py:59] DAG:
I 03-17 21:26:18 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 21:26:18 controller.py:59]   resources: {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}]
I 03-17 21:26:18 controller.py:195] Submitted managed job 490 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-21-26-18-264770_compute-text-vectors_490-0
I 03-17 21:26:18 controller.py:199] Started monitoring.
I 03-17 21:26:18 state.py:451] Launching the spot cluster...
I 03-17 21:26:18 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 21:26:19 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:26:19 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:26:19 optimizer.py:749] 
I 03-17 21:26:19 optimizer.py:884] Considered resources (1 node):
I 03-17 21:26:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:26:19 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:26:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:26:19 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 21:26:19 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:26:19 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:26:19 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
[2mW 03-17 21:36:01 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1a.[0m
W 03-17 21:36:03 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:36:03 cloud_vm_ray_backend.py:2097] 
W 03-17 21:36:03 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:36:03 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:36:03 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:36:03 optimizer.py:749] 
I 03-17 21:36:03 optimizer.py:884] Considered resources (1 node):
I 03-17 21:36:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:36:03 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:36:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:36:03 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 21:36:03 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:36:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:36:03 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 21:40:30 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:40:31 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:40:31 cloud_vm_ray_backend.py:2097] 
W 03-17 21:40:31 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:40:32 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:40:32 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:40:32 optimizer.py:749] 
I 03-17 21:40:32 optimizer.py:884] Considered resources (1 node):
I 03-17 21:40:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:40:32 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:40:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:40:32 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 21:40:32 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:40:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:40:32 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 21:46:28 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:46:29 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:46:29 cloud_vm_ray_backend.py:2097] 
W 03-17 21:46:29 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:46:30 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:46:30 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:46:30 optimizer.py:749] 
I 03-17 21:46:30 optimizer.py:884] Considered resources (1 node):
I 03-17 21:46:30 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:46:30 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:46:30 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:46:30 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 21:46:30 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:46:30 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:46:30 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
[2mW 03-17 21:54:04 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:54:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:54:05 cloud_vm_ray_backend.py:2097] 
W 03-17 21:54:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:54:06 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:54:06 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:54:06 optimizer.py:749] 
I 03-17 21:54:06 optimizer.py:884] Considered resources (1 node):
I 03-17 21:54:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:54:06 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:54:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:54:06 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 21:54:06 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:54:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:54:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 21:56:56 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-1a, us-east-1c, us-east-1d.[0m
W 03-17 21:56:56 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:56:56 cloud_vm_ray_backend.py:2097] 
W 03-17 21:56:56 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:56:57 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:56:57 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:56:57 optimizer.py:749] 
I 03-17 21:56:57 optimizer.py:884] Considered resources (1 node):
I 03-17 21:56:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:56:57 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:56:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:56:57 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 21:56:57 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:56:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:56:57 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 21:57:04 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 21:57:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:05 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:05 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:05 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:05 optimizer.py:749] 
I 03-17 21:57:05 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:05 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:05 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 21:57:05 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:05 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 21:57:12 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:57:12 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:12 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:12 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:12 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:12 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:12 optimizer.py:749] 
I 03-17 21:57:12 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:12 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:57:12 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 21:57:12 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:57:12 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 21:57:12 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 21:57:12 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:13 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 21:57:18 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:57:18 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:18 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:18 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:18 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:18 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:18 optimizer.py:749] 
I 03-17 21:57:18 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:18 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:18 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:18 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:18 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 21:57:18 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:18 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:18 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 21:57:19 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 21:57:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:20 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:20 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:20 optimizer.py:749] 
I 03-17 21:57:20 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:57:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 21:57:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 21:57:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 21:57:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 21:57:20 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 21:57:25 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:57:25 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:25 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:25 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:25 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:25 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:25 optimizer.py:749] 
I 03-17 21:57:25 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:25 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:25 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 21:57:25 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:25 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:25 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 21:57:40 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:57:41 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:41 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:41 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:41 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:41 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:41 optimizer.py:749] 
I 03-17 21:57:41 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:41 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:41 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 21:57:41 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:41 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 21:57:48 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3b.[0m
W 03-17 21:57:49 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:57:49 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:49 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:49 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:49 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:57:49 optimizer.py:749] 
I 03-17 21:57:49 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:49 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:49 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 21:57:49 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:49 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 21:58:00 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:58:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:58:01 cloud_vm_ray_backend.py:2097] 
W 03-17 21:58:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:58:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:58:01 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:58:01 optimizer.py:749] 
I 03-17 21:58:01 optimizer.py:884] Considered resources (1 node):
I 03-17 21:58:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:58:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 21:58:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:58:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:58:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 21:58:08 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 21:58:09 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:58:09 cloud_vm_ray_backend.py:2097] 
W 03-17 21:58:09 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:58:10 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:58:10 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:58:10 optimizer.py:749] 
I 03-17 21:58:10 optimizer.py:884] Considered resources (1 node):
I 03-17 21:58:10 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:58:10 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:58:10 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:58:10 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 21:58:10 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:58:10 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:58:10 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
[2mW 03-17 21:58:21 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2a.[0m
W 03-17 21:58:22 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 21:58:22 cloud_vm_ray_backend.py:2097] 
W 03-17 21:58:22 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:58:22 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:58:22 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 21:58:22 optimizer.py:749] 
I 03-17 21:58:22 optimizer.py:884] Considered resources (1 node):
I 03-17 21:58:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:22 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:58:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:58:22 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1d    0.37       [32m   âœ”[0m     
I 03-17 21:58:22 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:58:22 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:58:22 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1d).
[2mW 03-17 22:03:27 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:03:28 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1d for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:03:28 cloud_vm_ray_backend.py:2097] 
W 03-17 22:03:28 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:03:28 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:03:28 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:03:28 optimizer.py:749] 
I 03-17 22:03:28 optimizer.py:884] Considered resources (1 node):
I 03-17 22:03:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:03:28 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:03:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:03:28 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2b    0.37       [32m   âœ”[0m     
I 03-17 22:03:28 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:03:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:03:28 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2b).
[2mW 03-17 22:03:36 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2a, us-west-2c, us-west-2d.[0m
W 03-17 22:03:37 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:03:37 cloud_vm_ray_backend.py:2097] 
W 03-17 22:03:37 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:03:37 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:03:37 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:03:37 optimizer.py:749] 
I 03-17 22:03:37 optimizer.py:884] Considered resources (1 node):
I 03-17 22:03:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:03:37 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:03:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:03:37 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1b   0.37       [32m   âœ”[0m     
I 03-17 22:03:37 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:03:37 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:03:37 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1b).
[2mW 03-17 22:03:48 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:03:49 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:03:49 cloud_vm_ray_backend.py:2097] 
W 03-17 22:03:49 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:03:49 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:03:49 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:03:49 optimizer.py:749] 
I 03-17 22:03:49 optimizer.py:884] Considered resources (1 node):
I 03-17 22:03:49 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:03:49 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:03:49 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:03:49 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2a   0.38       [32m   âœ”[0m     
I 03-17 22:03:49 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:03:49 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:03:49 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a).
[2mW 03-17 22:04:00 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2c.[0m
W 03-17 22:04:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:04:01 cloud_vm_ray_backend.py:2097] 
W 03-17 22:04:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:04:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:04:01 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:04:01 optimizer.py:749] 
I 03-17 22:04:01 optimizer.py:884] Considered resources (1 node):
I 03-17 22:04:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:04:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3b    0.38       [32m   âœ”[0m     
I 03-17 22:04:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:04:01 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:04:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b).
[2mW 03-17 22:04:09 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:04:10 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:04:10 cloud_vm_ray_backend.py:2097] 
W 03-17 22:04:10 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:04:10 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:04:10 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:04:10 optimizer.py:749] 
I 03-17 22:04:10 optimizer.py:884] Considered resources (1 node):
I 03-17 22:04:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:10 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:04:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:10 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1a    0.39       [32m   âœ”[0m     
I 03-17 22:04:10 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:04:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:04:11 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a).
[2mW 03-17 22:06:41 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:06:41 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:06:41 cloud_vm_ray_backend.py:2097] 
W 03-17 22:06:41 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:06:41 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:06:41 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:06:41 optimizer.py:749] 
I 03-17 22:06:41 optimizer.py:884] Considered resources (1 node):
I 03-17 22:06:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:06:41 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:06:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:06:41 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2a    0.40       [32m   âœ”[0m     
I 03-17 22:06:41 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:06:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:06:41 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a).
W 03-17 22:17:23 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:17:23 cloud_vm_ray_backend.py:2097] 
W 03-17 22:17:23 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:17:23 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:17:23 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:17:23 optimizer.py:749] 
I 03-17 22:17:23 optimizer.py:884] Considered resources (1 node):
I 03-17 22:17:23 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:17:23 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:17:23 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:17:23 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1b   0.40       [32m   âœ”[0m     
I 03-17 22:17:23 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:17:23 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:17:23 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1b).
[2mW 03-17 22:17:32 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:17:33 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1b for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:17:33 cloud_vm_ray_backend.py:2097] 
W 03-17 22:17:33 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:17:33 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:17:33 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:17:33 optimizer.py:749] 
I 03-17 22:17:33 optimizer.py:884] Considered resources (1 node):
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:33 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:17:33 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2d    0.41       [32m   âœ”[0m     
I 03-17 22:17:33 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:17:33 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:17:33 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2d).
[2mW 03-17 22:18:00 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2d for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:18:01 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:01 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:01 optimizer.py:749] 
I 03-17 22:18:01 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:01 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:18:01 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1a   0.41       [32m   âœ”[0m     
I 03-17 22:18:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:18:01 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:02 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1a).
[2mW 03-17 22:18:11 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:12 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:18:12 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:12 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:12 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:12 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:18:12 optimizer.py:749] 
I 03-17 22:18:12 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:12 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:12 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2c    0.41       [32m   âœ”[0m     
I 03-17 22:18:12 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:12 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2c).
[2mW 03-17 22:18:46 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:47 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2c for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:18:47 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:47 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:47 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:47 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:18:47 optimizer.py:749] 
I 03-17 22:18:47 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:47 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:18:47 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:18:47 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:18:47 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1a   0.46       [32m   âœ”[0m     
I 03-17 22:18:47 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:18:47 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:47 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1a).
[2mW 03-17 22:18:59 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:19:00 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-1a for {AWS(cpus=4, {'L4': 1}), AWS([Spot], cpus=4, {'L4': 1})}. 
W 03-17 22:19:00 cloud_vm_ray_backend.py:2097] 
W 03-17 22:19:00 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:00 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:19:00 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:19:00 optimizer.py:749] 
I 03-17 22:19:00 optimizer.py:884] Considered resources (1 node):
I 03-17 22:19:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:19:00 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:19:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:19:00 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1c   0.48       [32m   âœ”[0m     
I 03-17 22:19:00 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:19:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:00 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1c).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:28 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:54 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-490.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 22:19:54 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:55 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-3mm3ndk8/job-3mm3ndk8/workdir -> ~/sky_workdir[0m
I 03-17 22:19:58 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:58 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final-2 -> /output[0m
I 03-17 22:20:01 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-21-26-18-861409/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 22:20:04 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 22:20:05 cloud_vm_ray_backend.py:3546] Multiple resources are specified for the task, using: AWS([Spot], cpus=4, {'L4': 1})
I 03-17 22:20:07 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 22:20:07 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 22:20:11 utils.py:131] === Checking the job status... ===
I 03-17 22:20:12 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:20:12 utils.py:140] ==================================
I 03-17 22:20:13 state.py:471] Job started.
I 03-17 22:20:33 utils.py:131] === Checking the job status... ===
I 03-17 22:20:34 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:20:34 utils.py:140] ==================================
I 03-17 22:20:54 utils.py:131] === Checking the job status... ===
I 03-17 22:20:55 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:20:55 utils.py:140] ==================================
I 03-17 22:21:15 utils.py:131] === Checking the job status... ===
I 03-17 22:21:16 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:21:16 utils.py:140] ==================================
I 03-17 22:21:36 utils.py:131] === Checking the job status... ===
I 03-17 22:21:37 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:21:37 utils.py:140] ==================================
I 03-17 22:21:57 utils.py:131] === Checking the job status... ===
I 03-17 22:21:58 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:21:58 utils.py:140] ==================================
I 03-17 22:22:19 utils.py:131] === Checking the job status... ===
I 03-17 22:22:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:22:20 utils.py:140] ==================================
I 03-17 22:22:40 utils.py:131] === Checking the job status... ===
I 03-17 22:22:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:22:41 utils.py:140] ==================================
I 03-17 22:23:01 utils.py:131] === Checking the job status... ===
I 03-17 22:23:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:02 utils.py:140] ==================================
I 03-17 22:23:22 utils.py:131] === Checking the job status... ===
I 03-17 22:23:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:23 utils.py:140] ==================================
I 03-17 22:23:43 utils.py:131] === Checking the job status... ===
I 03-17 22:23:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:23:44 utils.py:140] ==================================
I 03-17 22:24:05 utils.py:131] === Checking the job status... ===
I 03-17 22:24:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:24:05 utils.py:140] ==================================
I 03-17 22:24:26 utils.py:131] === Checking the job status... ===
I 03-17 22:24:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:24:27 utils.py:140] ==================================
I 03-17 22:24:47 utils.py:131] === Checking the job status... ===
I 03-17 22:24:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:24:48 utils.py:140] ==================================
I 03-17 22:25:08 utils.py:131] === Checking the job status... ===
I 03-17 22:25:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:09 utils.py:140] ==================================
I 03-17 22:25:29 utils.py:131] === Checking the job status... ===
I 03-17 22:25:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:30 utils.py:140] ==================================
I 03-17 22:25:50 utils.py:131] === Checking the job status... ===
I 03-17 22:25:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:25:51 utils.py:140] ==================================
I 03-17 22:26:11 utils.py:131] === Checking the job status... ===
I 03-17 22:26:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:12 utils.py:140] ==================================
I 03-17 22:26:32 utils.py:131] === Checking the job status... ===
I 03-17 22:26:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:33 utils.py:140] ==================================
I 03-17 22:26:53 utils.py:131] === Checking the job status... ===
I 03-17 22:26:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:26:54 utils.py:140] ==================================
I 03-17 22:27:14 utils.py:131] === Checking the job status... ===
I 03-17 22:27:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:15 utils.py:140] ==================================
I 03-17 22:27:35 utils.py:131] === Checking the job status... ===
I 03-17 22:27:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:36 utils.py:140] ==================================
I 03-17 22:27:56 utils.py:131] === Checking the job status... ===
I 03-17 22:27:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:27:57 utils.py:140] ==================================
I 03-17 22:28:17 utils.py:131] === Checking the job status... ===
I 03-17 22:28:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:28:18 utils.py:140] ==================================
I 03-17 22:28:38 utils.py:131] === Checking the job status... ===
I 03-17 22:28:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:28:39 utils.py:140] ==================================
I 03-17 22:28:59 utils.py:131] === Checking the job status... ===
I 03-17 22:29:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:00 utils.py:140] ==================================
I 03-17 22:29:20 utils.py:131] === Checking the job status... ===
I 03-17 22:29:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:21 utils.py:140] ==================================
I 03-17 22:29:41 utils.py:131] === Checking the job status... ===
I 03-17 22:29:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:29:42 utils.py:140] ==================================
I 03-17 22:30:02 utils.py:131] === Checking the job status... ===
I 03-17 22:30:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:03 utils.py:140] ==================================
I 03-17 22:30:23 utils.py:131] === Checking the job status... ===
I 03-17 22:30:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:24 utils.py:140] ==================================
I 03-17 22:30:44 utils.py:131] === Checking the job status... ===
I 03-17 22:30:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:30:45 utils.py:140] ==================================
I 03-17 22:31:05 utils.py:131] === Checking the job status... ===
I 03-17 22:31:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:06 utils.py:140] ==================================
I 03-17 22:31:26 utils.py:131] === Checking the job status... ===
I 03-17 22:31:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:27 utils.py:140] ==================================
I 03-17 22:31:47 utils.py:131] === Checking the job status... ===
I 03-17 22:31:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:31:48 utils.py:140] ==================================
I 03-17 22:32:08 utils.py:131] === Checking the job status... ===
I 03-17 22:32:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:32:09 utils.py:140] ==================================
I 03-17 22:32:29 utils.py:131] === Checking the job status... ===
I 03-17 22:32:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:32:30 utils.py:140] ==================================
I 03-17 22:32:50 utils.py:131] === Checking the job status... ===
I 03-17 22:32:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:32:51 utils.py:140] ==================================
I 03-17 22:33:11 utils.py:131] === Checking the job status... ===
I 03-17 22:33:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:12 utils.py:140] ==================================
I 03-17 22:33:32 utils.py:131] === Checking the job status... ===
I 03-17 22:33:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:33 utils.py:140] ==================================
I 03-17 22:33:53 utils.py:131] === Checking the job status... ===
I 03-17 22:33:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:33:54 utils.py:140] ==================================
I 03-17 22:34:14 utils.py:131] === Checking the job status... ===
I 03-17 22:34:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:15 utils.py:140] ==================================
I 03-17 22:34:35 utils.py:131] === Checking the job status... ===
I 03-17 22:34:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:36 utils.py:140] ==================================
I 03-17 22:34:56 utils.py:131] === Checking the job status... ===
I 03-17 22:34:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:34:57 utils.py:140] ==================================
I 03-17 22:35:17 utils.py:131] === Checking the job status... ===
I 03-17 22:35:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:35:18 utils.py:140] ==================================
I 03-17 22:35:38 utils.py:131] === Checking the job status... ===
I 03-17 22:35:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:35:39 utils.py:140] ==================================
I 03-17 22:35:59 utils.py:131] === Checking the job status... ===
I 03-17 22:36:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:00 utils.py:140] ==================================
I 03-17 22:36:20 utils.py:131] === Checking the job status... ===
I 03-17 22:36:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:21 utils.py:140] ==================================
I 03-17 22:36:41 utils.py:131] === Checking the job status... ===
I 03-17 22:36:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:36:42 utils.py:140] ==================================
I 03-17 22:37:02 utils.py:131] === Checking the job status... ===
I 03-17 22:37:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:37:03 utils.py:140] ==================================
I 03-17 22:37:23 utils.py:131] === Checking the job status... ===
I 03-17 22:37:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:37:24 utils.py:140] ==================================
I 03-17 22:37:44 utils.py:131] === Checking the job status... ===
I 03-17 22:37:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:37:45 utils.py:140] ==================================
I 03-17 22:38:05 utils.py:131] === Checking the job status... ===
I 03-17 22:38:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:38:06 utils.py:140] ==================================
I 03-17 22:38:27 utils.py:131] === Checking the job status... ===
I 03-17 22:38:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:38:27 utils.py:140] ==================================
I 03-17 22:38:48 utils.py:131] === Checking the job status... ===
I 03-17 22:38:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:38:49 utils.py:140] ==================================
I 03-17 22:39:09 utils.py:131] === Checking the job status... ===
I 03-17 22:39:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:10 utils.py:140] ==================================
I 03-17 22:39:30 utils.py:131] === Checking the job status... ===
I 03-17 22:39:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:31 utils.py:140] ==================================
I 03-17 22:39:51 utils.py:131] === Checking the job status... ===
I 03-17 22:39:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:39:52 utils.py:140] ==================================
I 03-17 22:40:12 utils.py:131] === Checking the job status... ===
I 03-17 22:40:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:13 utils.py:140] ==================================
I 03-17 22:40:33 utils.py:131] === Checking the job status... ===
I 03-17 22:40:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:34 utils.py:140] ==================================
I 03-17 22:40:54 utils.py:131] === Checking the job status... ===
I 03-17 22:40:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:40:55 utils.py:140] ==================================
I 03-17 22:41:15 utils.py:131] === Checking the job status... ===
I 03-17 22:41:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:41:16 utils.py:140] ==================================
I 03-17 22:41:36 utils.py:131] === Checking the job status... ===
I 03-17 22:41:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:41:37 utils.py:140] ==================================
I 03-17 22:41:57 utils.py:131] === Checking the job status... ===
I 03-17 22:41:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:41:58 utils.py:140] ==================================
I 03-17 22:42:18 utils.py:131] === Checking the job status... ===
I 03-17 22:42:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:42:19 utils.py:140] ==================================
I 03-17 22:42:39 utils.py:131] === Checking the job status... ===
I 03-17 22:42:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:42:40 utils.py:140] ==================================
I 03-17 22:43:00 utils.py:131] === Checking the job status... ===
I 03-17 22:43:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:01 utils.py:140] ==================================
I 03-17 22:43:21 utils.py:131] === Checking the job status... ===
I 03-17 22:43:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:22 utils.py:140] ==================================
I 03-17 22:43:42 utils.py:131] === Checking the job status... ===
I 03-17 22:43:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:43:43 utils.py:140] ==================================
I 03-17 22:44:03 utils.py:131] === Checking the job status... ===
I 03-17 22:44:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:04 utils.py:140] ==================================
I 03-17 22:44:24 utils.py:131] === Checking the job status... ===
I 03-17 22:44:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:25 utils.py:140] ==================================
I 03-17 22:44:45 utils.py:131] === Checking the job status... ===
I 03-17 22:44:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:44:46 utils.py:140] ==================================
I 03-17 22:45:06 utils.py:131] === Checking the job status... ===
I 03-17 22:45:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:07 utils.py:140] ==================================
I 03-17 22:45:27 utils.py:131] === Checking the job status... ===
I 03-17 22:45:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:28 utils.py:140] ==================================
I 03-17 22:45:48 utils.py:131] === Checking the job status... ===
I 03-17 22:45:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:45:49 utils.py:140] ==================================
I 03-17 22:46:09 utils.py:131] === Checking the job status... ===
I 03-17 22:46:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:10 utils.py:140] ==================================
I 03-17 22:46:30 utils.py:131] === Checking the job status... ===
I 03-17 22:46:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:31 utils.py:140] ==================================
I 03-17 22:46:51 utils.py:131] === Checking the job status... ===
I 03-17 22:46:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:46:52 utils.py:140] ==================================
I 03-17 22:47:12 utils.py:131] === Checking the job status... ===
I 03-17 22:47:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:13 utils.py:140] ==================================
I 03-17 22:47:33 utils.py:131] === Checking the job status... ===
I 03-17 22:47:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:34 utils.py:140] ==================================
I 03-17 22:47:54 utils.py:131] === Checking the job status... ===
I 03-17 22:47:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:47:55 utils.py:140] ==================================
I 03-17 22:48:15 utils.py:131] === Checking the job status... ===
I 03-17 22:48:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:48:16 utils.py:140] ==================================
I 03-17 22:48:36 utils.py:131] === Checking the job status... ===
I 03-17 22:48:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:48:37 utils.py:140] ==================================
I 03-17 22:48:57 utils.py:131] === Checking the job status... ===
I 03-17 22:48:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:48:58 utils.py:140] ==================================
I 03-17 22:49:18 utils.py:131] === Checking the job status... ===
I 03-17 22:49:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:49:19 utils.py:140] ==================================
I 03-17 22:49:39 utils.py:131] === Checking the job status... ===
I 03-17 22:49:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:49:40 utils.py:140] ==================================
I 03-17 22:50:00 utils.py:131] === Checking the job status... ===
I 03-17 22:50:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:01 utils.py:140] ==================================
I 03-17 22:50:21 utils.py:131] === Checking the job status... ===
I 03-17 22:50:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:22 utils.py:140] ==================================
I 03-17 22:50:42 utils.py:131] === Checking the job status... ===
I 03-17 22:50:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:50:43 utils.py:140] ==================================
I 03-17 22:51:03 utils.py:131] === Checking the job status... ===
I 03-17 22:51:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:04 utils.py:140] ==================================
I 03-17 22:51:24 utils.py:131] === Checking the job status... ===
I 03-17 22:51:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:25 utils.py:140] ==================================
I 03-17 22:51:45 utils.py:131] === Checking the job status... ===
I 03-17 22:51:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:51:46 utils.py:140] ==================================
I 03-17 22:52:06 utils.py:131] === Checking the job status... ===
I 03-17 22:52:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:07 utils.py:140] ==================================
I 03-17 22:52:27 utils.py:131] === Checking the job status... ===
I 03-17 22:52:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:28 utils.py:140] ==================================
I 03-17 22:52:48 utils.py:131] === Checking the job status... ===
I 03-17 22:52:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:52:49 utils.py:140] ==================================
I 03-17 22:53:10 utils.py:131] === Checking the job status... ===
I 03-17 22:53:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:10 utils.py:140] ==================================
I 03-17 22:53:31 utils.py:131] === Checking the job status... ===
I 03-17 22:53:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:31 utils.py:140] ==================================
I 03-17 22:53:52 utils.py:131] === Checking the job status... ===
I 03-17 22:53:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:53:53 utils.py:140] ==================================
I 03-17 22:54:13 utils.py:131] === Checking the job status... ===
I 03-17 22:54:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:14 utils.py:140] ==================================
I 03-17 22:54:34 utils.py:131] === Checking the job status... ===
I 03-17 22:54:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:35 utils.py:140] ==================================
I 03-17 22:54:55 utils.py:131] === Checking the job status... ===
I 03-17 22:54:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:56 utils.py:140] ==================================
I 03-17 22:55:16 utils.py:131] === Checking the job status... ===
I 03-17 22:55:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:17 utils.py:140] ==================================
I 03-17 22:55:37 utils.py:131] === Checking the job status... ===
I 03-17 22:55:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:38 utils.py:140] ==================================
I 03-17 22:55:58 utils.py:131] === Checking the job status... ===
I 03-17 22:55:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:59 utils.py:140] ==================================
I 03-17 22:56:19 utils.py:131] === Checking the job status... ===
I 03-17 22:56:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:20 utils.py:140] ==================================
I 03-17 22:56:40 utils.py:131] === Checking the job status... ===
I 03-17 22:56:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:41 utils.py:140] ==================================
I 03-17 22:57:01 utils.py:131] === Checking the job status... ===
I 03-17 22:57:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:02 utils.py:140] ==================================
I 03-17 22:57:22 utils.py:131] === Checking the job status... ===
I 03-17 22:57:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:23 utils.py:140] ==================================
I 03-17 22:57:43 utils.py:131] === Checking the job status... ===
I 03-17 22:57:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:44 utils.py:140] ==================================
I 03-17 22:58:04 utils.py:131] === Checking the job status... ===
I 03-17 22:58:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:05 utils.py:140] ==================================
I 03-17 22:58:25 utils.py:131] === Checking the job status... ===
I 03-17 22:58:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:26 utils.py:140] ==================================
I 03-17 22:58:46 utils.py:131] === Checking the job status... ===
I 03-17 22:58:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:47 utils.py:140] ==================================
I 03-17 22:59:07 utils.py:131] === Checking the job status... ===
I 03-17 22:59:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:08 utils.py:140] ==================================
I 03-17 22:59:28 utils.py:131] === Checking the job status... ===
I 03-17 22:59:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:29 utils.py:140] ==================================
I 03-17 22:59:49 utils.py:131] === Checking the job status... ===
I 03-17 22:59:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:50 utils.py:140] ==================================
I 03-17 23:00:10 utils.py:131] === Checking the job status... ===
I 03-17 23:00:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:11 utils.py:140] ==================================
I 03-17 23:00:31 utils.py:131] === Checking the job status... ===
I 03-17 23:00:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:32 utils.py:140] ==================================
I 03-17 23:00:52 utils.py:131] === Checking the job status... ===
I 03-17 23:00:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:53 utils.py:140] ==================================
I 03-17 23:01:13 utils.py:131] === Checking the job status... ===
I 03-17 23:01:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:14 utils.py:140] ==================================
I 03-17 23:01:34 utils.py:131] === Checking the job status... ===
I 03-17 23:01:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:35 utils.py:140] ==================================
I 03-17 23:01:55 utils.py:131] === Checking the job status... ===
I 03-17 23:01:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:56 utils.py:140] ==================================
I 03-17 23:02:16 utils.py:131] === Checking the job status... ===
I 03-17 23:02:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:17 utils.py:140] ==================================
I 03-17 23:02:37 utils.py:131] === Checking the job status... ===
I 03-17 23:02:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:38 utils.py:140] ==================================
I 03-17 23:02:58 utils.py:131] === Checking the job status... ===
I 03-17 23:02:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:59 utils.py:140] ==================================
I 03-17 23:03:19 utils.py:131] === Checking the job status... ===
I 03-17 23:03:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:20 utils.py:140] ==================================
I 03-17 23:03:40 utils.py:131] === Checking the job status... ===
I 03-17 23:03:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:41 utils.py:140] ==================================
I 03-17 23:04:01 utils.py:131] === Checking the job status... ===
I 03-17 23:04:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:02 utils.py:140] ==================================
I 03-17 23:04:22 utils.py:131] === Checking the job status... ===
I 03-17 23:04:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:23 utils.py:140] ==================================
I 03-17 23:04:43 utils.py:131] === Checking the job status... ===
I 03-17 23:04:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:44 utils.py:140] ==================================
I 03-17 23:05:04 utils.py:131] === Checking the job status... ===
I 03-17 23:05:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:05 utils.py:140] ==================================
I 03-17 23:05:25 utils.py:131] === Checking the job status... ===
I 03-17 23:05:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:26 utils.py:140] ==================================
I 03-17 23:05:46 utils.py:131] === Checking the job status... ===
I 03-17 23:05:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:47 utils.py:140] ==================================
I 03-17 23:06:07 utils.py:131] === Checking the job status... ===
I 03-17 23:06:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:08 utils.py:140] ==================================
I 03-17 23:06:28 utils.py:131] === Checking the job status... ===
I 03-17 23:06:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:29 utils.py:140] ==================================
I 03-17 23:06:49 utils.py:131] === Checking the job status... ===
I 03-17 23:06:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:50 utils.py:140] ==================================
I 03-17 23:07:10 utils.py:131] === Checking the job status... ===
I 03-17 23:07:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:11 utils.py:140] ==================================
I 03-17 23:07:31 utils.py:131] === Checking the job status... ===
I 03-17 23:07:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:32 utils.py:140] ==================================
I 03-17 23:07:52 utils.py:131] === Checking the job status... ===
I 03-17 23:07:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:53 utils.py:140] ==================================
I 03-17 23:08:13 utils.py:131] === Checking the job status... ===
I 03-17 23:08:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:14 utils.py:140] ==================================
I 03-17 23:08:34 utils.py:131] === Checking the job status... ===
I 03-17 23:08:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:35 utils.py:140] ==================================
I 03-17 23:08:55 utils.py:131] === Checking the job status... ===
I 03-17 23:08:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:56 utils.py:140] ==================================
I 03-17 23:09:16 utils.py:131] === Checking the job status... ===
I 03-17 23:09:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:17 utils.py:140] ==================================
I 03-17 23:09:37 utils.py:131] === Checking the job status... ===
I 03-17 23:09:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:38 utils.py:140] ==================================
I 03-17 23:09:58 utils.py:131] === Checking the job status... ===
I 03-17 23:09:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:59 utils.py:140] ==================================
I 03-17 23:10:19 utils.py:131] === Checking the job status... ===
I 03-17 23:10:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:20 utils.py:140] ==================================
I 03-17 23:10:41 utils.py:131] === Checking the job status... ===
I 03-17 23:10:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:41 utils.py:140] ==================================
I 03-17 23:11:02 utils.py:131] === Checking the job status... ===
I 03-17 23:11:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:02 utils.py:140] ==================================
I 03-17 23:11:23 utils.py:131] === Checking the job status... ===
I 03-17 23:11:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:24 utils.py:140] ==================================
I 03-17 23:11:44 utils.py:131] === Checking the job status... ===
I 03-17 23:11:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:45 utils.py:140] ==================================
I 03-17 23:12:05 utils.py:131] === Checking the job status... ===
I 03-17 23:12:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:06 utils.py:140] ==================================
I 03-17 23:12:26 utils.py:131] === Checking the job status... ===
I 03-17 23:12:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:27 utils.py:140] ==================================
I 03-17 23:12:47 utils.py:131] === Checking the job status... ===
I 03-17 23:12:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:48 utils.py:140] ==================================
I 03-17 23:13:08 utils.py:131] === Checking the job status... ===
I 03-17 23:13:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:09 utils.py:140] ==================================
I 03-17 23:13:29 utils.py:131] === Checking the job status... ===
I 03-17 23:13:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:30 utils.py:140] ==================================
I 03-17 23:13:50 utils.py:131] === Checking the job status... ===
I 03-17 23:13:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:51 utils.py:140] ==================================
I 03-17 23:14:11 utils.py:131] === Checking the job status... ===
I 03-17 23:14:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:12 utils.py:140] ==================================
I 03-17 23:14:32 utils.py:131] === Checking the job status... ===
I 03-17 23:14:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:33 utils.py:140] ==================================
I 03-17 23:14:53 utils.py:131] === Checking the job status... ===
I 03-17 23:14:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:54 utils.py:140] ==================================
I 03-17 23:15:14 utils.py:131] === Checking the job status... ===
I 03-17 23:15:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:15 utils.py:140] ==================================
I 03-17 23:15:35 utils.py:131] === Checking the job status... ===
I 03-17 23:15:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:36 utils.py:140] ==================================
I 03-17 23:15:56 utils.py:131] === Checking the job status... ===
I 03-17 23:15:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:57 utils.py:140] ==================================
I 03-17 23:16:17 utils.py:131] === Checking the job status... ===
I 03-17 23:16:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:18 utils.py:140] ==================================
I 03-17 23:16:38 utils.py:131] === Checking the job status... ===
I 03-17 23:16:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:39 utils.py:140] ==================================
I 03-17 23:16:59 utils.py:131] === Checking the job status... ===
I 03-17 23:17:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:00 utils.py:140] ==================================
I 03-17 23:17:20 utils.py:131] === Checking the job status... ===
I 03-17 23:17:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:21 utils.py:140] ==================================
I 03-17 23:17:41 utils.py:131] === Checking the job status... ===
I 03-17 23:17:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:42 utils.py:140] ==================================
I 03-17 23:18:02 utils.py:131] === Checking the job status... ===
I 03-17 23:18:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:03 utils.py:140] ==================================
I 03-17 23:18:23 utils.py:131] === Checking the job status... ===
I 03-17 23:18:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:24 utils.py:140] ==================================
I 03-17 23:18:44 utils.py:131] === Checking the job status... ===
I 03-17 23:18:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:45 utils.py:140] ==================================
I 03-17 23:19:05 utils.py:131] === Checking the job status... ===
I 03-17 23:19:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:06 utils.py:140] ==================================
I 03-17 23:19:26 utils.py:131] === Checking the job status... ===
I 03-17 23:19:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:27 utils.py:140] ==================================
I 03-17 23:19:47 utils.py:131] === Checking the job status... ===
I 03-17 23:19:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:48 utils.py:140] ==================================
I 03-17 23:20:08 utils.py:131] === Checking the job status... ===
I 03-17 23:20:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:09 utils.py:140] ==================================
I 03-17 23:20:29 utils.py:131] === Checking the job status... ===
I 03-17 23:20:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:30 utils.py:140] ==================================
I 03-17 23:20:50 utils.py:131] === Checking the job status... ===
I 03-17 23:20:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:51 utils.py:140] ==================================
I 03-17 23:21:11 utils.py:131] === Checking the job status... ===
I 03-17 23:21:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:12 utils.py:140] ==================================
I 03-17 23:21:32 utils.py:131] === Checking the job status... ===
I 03-17 23:21:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:33 utils.py:140] ==================================
I 03-17 23:21:53 utils.py:131] === Checking the job status... ===
I 03-17 23:21:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:54 utils.py:140] ==================================
I 03-17 23:22:14 utils.py:131] === Checking the job status... ===
I 03-17 23:22:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:15 utils.py:140] ==================================
I 03-17 23:22:35 utils.py:131] === Checking the job status... ===
I 03-17 23:22:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:36 utils.py:140] ==================================
I 03-17 23:22:56 utils.py:131] === Checking the job status... ===
I 03-17 23:22:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:57 utils.py:140] ==================================
I 03-17 23:23:17 utils.py:131] === Checking the job status... ===
I 03-17 23:23:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:18 utils.py:140] ==================================
I 03-17 23:23:38 utils.py:131] === Checking the job status... ===
I 03-17 23:23:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:39 utils.py:140] ==================================
I 03-17 23:23:59 utils.py:131] === Checking the job status... ===
I 03-17 23:24:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:00 utils.py:140] ==================================
I 03-17 23:24:20 utils.py:131] === Checking the job status... ===
I 03-17 23:24:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:21 utils.py:140] ==================================
I 03-17 23:24:41 utils.py:131] === Checking the job status... ===
I 03-17 23:24:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:42 utils.py:140] ==================================
I 03-17 23:25:02 utils.py:131] === Checking the job status... ===
I 03-17 23:25:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:03 utils.py:140] ==================================
I 03-17 23:25:23 utils.py:131] === Checking the job status... ===
I 03-17 23:25:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:24 utils.py:140] ==================================
I 03-17 23:25:44 utils.py:131] === Checking the job status... ===
I 03-17 23:25:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:45 utils.py:140] ==================================
I 03-17 23:26:05 utils.py:131] === Checking the job status... ===
I 03-17 23:26:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:06 utils.py:140] ==================================
I 03-17 23:26:26 utils.py:131] === Checking the job status... ===
I 03-17 23:26:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:27 utils.py:140] ==================================
I 03-17 23:26:47 utils.py:131] === Checking the job status... ===
I 03-17 23:26:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:48 utils.py:140] ==================================
I 03-17 23:27:08 utils.py:131] === Checking the job status... ===
I 03-17 23:27:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:09 utils.py:140] ==================================
I 03-17 23:27:30 utils.py:131] === Checking the job status... ===
I 03-17 23:27:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:30 utils.py:140] ==================================
I 03-17 23:27:51 utils.py:131] === Checking the job status... ===
I 03-17 23:27:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:51 utils.py:140] ==================================
I 03-17 23:28:12 utils.py:131] === Checking the job status... ===
I 03-17 23:28:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:12 utils.py:140] ==================================
I 03-17 23:28:33 utils.py:131] === Checking the job status... ===
I 03-17 23:28:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:34 utils.py:140] ==================================
I 03-17 23:28:54 utils.py:131] === Checking the job status... ===
I 03-17 23:28:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:55 utils.py:140] ==================================
I 03-17 23:29:15 utils.py:131] === Checking the job status... ===
I 03-17 23:29:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:16 utils.py:140] ==================================
I 03-17 23:29:36 utils.py:131] === Checking the job status... ===
I 03-17 23:29:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:37 utils.py:140] ==================================
I 03-17 23:29:57 utils.py:131] === Checking the job status... ===
I 03-17 23:29:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:58 utils.py:140] ==================================
I 03-17 23:30:18 utils.py:131] === Checking the job status... ===
I 03-17 23:30:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:19 utils.py:140] ==================================
I 03-17 23:30:39 utils.py:131] === Checking the job status... ===
I 03-17 23:30:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:40 utils.py:140] ==================================
I 03-17 23:31:00 utils.py:131] === Checking the job status... ===
I 03-17 23:31:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:01 utils.py:140] ==================================
I 03-17 23:31:21 utils.py:131] === Checking the job status... ===
I 03-17 23:31:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:22 utils.py:140] ==================================
I 03-17 23:31:42 utils.py:131] === Checking the job status... ===
I 03-17 23:31:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:43 utils.py:140] ==================================
I 03-17 23:32:03 utils.py:131] === Checking the job status... ===
I 03-17 23:32:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:04 utils.py:140] ==================================
I 03-17 23:32:24 utils.py:131] === Checking the job status... ===
I 03-17 23:32:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:25 utils.py:140] ==================================
I 03-17 23:32:45 utils.py:131] === Checking the job status... ===
I 03-17 23:32:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:46 utils.py:140] ==================================
I 03-17 23:33:06 utils.py:131] === Checking the job status... ===
I 03-17 23:33:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:07 utils.py:140] ==================================
I 03-17 23:33:27 utils.py:131] === Checking the job status... ===
I 03-17 23:33:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:28 utils.py:140] ==================================
I 03-17 23:33:48 utils.py:131] === Checking the job status... ===
I 03-17 23:33:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:49 utils.py:140] ==================================
I 03-17 23:34:09 utils.py:131] === Checking the job status... ===
I 03-17 23:34:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:10 utils.py:140] ==================================
I 03-17 23:34:30 utils.py:131] === Checking the job status... ===
I 03-17 23:34:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:31 utils.py:140] ==================================
I 03-17 23:34:51 utils.py:131] === Checking the job status... ===
I 03-17 23:34:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:52 utils.py:140] ==================================
I 03-17 23:35:12 utils.py:131] === Checking the job status... ===
I 03-17 23:35:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:13 utils.py:140] ==================================
I 03-17 23:35:33 utils.py:131] === Checking the job status... ===
I 03-17 23:35:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:34 utils.py:140] ==================================
I 03-17 23:35:54 utils.py:131] === Checking the job status... ===
E 03-17 23:35:54 subprocess_utils.py:151] ssh: connect to host 43.207.184.127 port 22: Connection refused
E 03-17 23:35:54 subprocess_utils.py:151] 
I 03-17 23:35:54 utils.py:139] Failed to connect to the cluster.
I 03-17 23:35:54 utils.py:140] ==================================
I 03-17 23:35:55 controller.py:295] Cluster is preempted or failed. Recovering...
I 03-17 23:35:55 state.py:501] === Recovering... ===
I 03-17 23:35:56 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-3mm3ndk8
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 23:35:56 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:35:56 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 23:35:56 optimizer.py:749] 
I 03-17 23:35:56 optimizer.py:884] Considered resources (1 node):
I 03-17 23:35:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:35:56 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:35:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:35:56 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 23:35:56 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:35:56 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:35:57 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
[2mW 03-17 23:36:09 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:36:10 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:36:10 cloud_vm_ray_backend.py:2097] 
W 03-17 23:36:10 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:36:10 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:36:10 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 23:36:10 optimizer.py:749] 
I 03-17 23:36:10 optimizer.py:884] Considered resources (1 node):
I 03-17 23:36:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:36:10 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:36:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:36:10 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 23:36:10 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:36:10 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:36:11 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 23:36:30 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:36:31 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:36:31 cloud_vm_ray_backend.py:2097] 
W 03-17 23:36:31 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:36:32 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:36:32 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:36:32 optimizer.py:749] 
I 03-17 23:36:32 optimizer.py:884] Considered resources (1 node):
I 03-17 23:36:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:36:32 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 23:36:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:36:32 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 23:36:32 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 23:36:32 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:36:32 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 23:36:44 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 23:36:46 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:36:46 cloud_vm_ray_backend.py:2097] 
W 03-17 23:36:46 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:36:46 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:36:46 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:36:46 optimizer.py:749] 
I 03-17 23:36:46 optimizer.py:884] Considered resources (1 node):
I 03-17 23:36:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:36:46 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 23:36:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:36:46 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 23:36:46 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 23:36:46 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:36:46 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
[2mW 03-17 23:36:58 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
W 03-17 23:36:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:36:59 cloud_vm_ray_backend.py:2097] 
W 03-17 23:36:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:00 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:00 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:00 optimizer.py:749] 
I 03-17 23:37:00 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:00 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:00 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:37:00 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:00 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 23:37:00 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:37:00 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:00 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 23:37:11 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:37:11 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:37:11 cloud_vm_ray_backend.py:2097] 
W 03-17 23:37:11 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:11 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:11 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:11 optimizer.py:749] 
I 03-17 23:37:11 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:11 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:11 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:37:11 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:11 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 23:37:11 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:37:11 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:12 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 23:37:25 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 23:37:26 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:37:26 cloud_vm_ray_backend.py:2097] 
W 03-17 23:37:26 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:26 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:26 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:26 optimizer.py:749] 
I 03-17 23:37:26 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:26 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:37:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:26 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 23:37:26 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:37:26 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:26 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 23:37:51 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:37:51 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:37:51 cloud_vm_ray_backend.py:2097] 
W 03-17 23:37:51 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:51 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:51 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:51 optimizer.py:749] 
I 03-17 23:37:51 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 23:37:51 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 23:37:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 23:37:51 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 23:37:51 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 23:37:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:52 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 23:37:57 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:37:57 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:37:57 cloud_vm_ray_backend.py:2097] 
W 03-17 23:37:57 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:57 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:57 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:57 optimizer.py:749] 
I 03-17 23:37:57 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:57 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:37:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:37:57 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 23:37:57 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:37:57 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:57 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 23:37:58 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 23:37:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:37:59 cloud_vm_ray_backend.py:2097] 
W 03-17 23:37:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:37:59 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:37:59 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:37:59 optimizer.py:749] 
I 03-17 23:37:59 optimizer.py:884] Considered resources (1 node):
I 03-17 23:37:59 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 23:37:59 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 23:37:59 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 23:37:59 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 23:37:59 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 23:37:59 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:37:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 23:38:08 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:38:08 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:38:08 cloud_vm_ray_backend.py:2097] 
W 03-17 23:38:08 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:38:08 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:38:08 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:38:08 optimizer.py:749] 
I 03-17 23:38:08 optimizer.py:884] Considered resources (1 node):
I 03-17 23:38:08 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:08 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:38:08 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:08 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 23:38:08 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:38:08 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:38:08 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 23:38:23 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:38:23 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:38:23 cloud_vm_ray_backend.py:2097] 
W 03-17 23:38:23 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:38:24 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:38:24 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 23:38:24 optimizer.py:749] 
I 03-17 23:38:24 optimizer.py:884] Considered resources (1 node):
I 03-17 23:38:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:24 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:38:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:24 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 23:38:24 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:38:24 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:38:24 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 23:38:32 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 23:38:32 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:38:32 cloud_vm_ray_backend.py:2097] 
W 03-17 23:38:32 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:38:32 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:38:32 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:38:32 optimizer.py:749] 
I 03-17 23:38:32 optimizer.py:884] Considered resources (1 node):
I 03-17 23:38:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:32 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:38:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:32 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 23:38:32 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:38:32 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:38:33 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 23:38:43 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-north-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-north-1b.[0m
W 03-17 23:38:44 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:38:44 cloud_vm_ray_backend.py:2097] 
W 03-17 23:38:44 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:38:44 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:38:44 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:38:44 optimizer.py:749] 
I 03-17 23:38:44 optimizer.py:884] Considered resources (1 node):
I 03-17 23:38:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:44 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:38:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:38:44 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 23:38:44 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:38:44 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:38:45 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 23:38:51 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:38:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:38:52 cloud_vm_ray_backend.py:2097] 
W 03-17 23:38:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:38:53 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:38:53 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:38:53 optimizer.py:749] 
I 03-17 23:38:53 optimizer.py:884] Considered resources (1 node):
I 03-17 23:38:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:38:53 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 23:38:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:38:53 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 23:38:53 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 23:38:53 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:38:53 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
[2mW 03-17 23:39:03 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2a.[0m
W 03-17 23:39:04 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:39:04 cloud_vm_ray_backend.py:2097] 
W 03-17 23:39:04 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:39:05 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:39:05 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:39:05 optimizer.py:749] 
I 03-17 23:39:05 optimizer.py:884] Considered resources (1 node):
I 03-17 23:39:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:39:05 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:39:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:39:05 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1d    0.37       [32m   âœ”[0m     
I 03-17 23:39:05 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:39:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:39:05 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1d).
[2mW 03-17 23:39:12 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:39:12 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:39:12 cloud_vm_ray_backend.py:2097] 
W 03-17 23:39:12 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:39:12 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:39:12 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:39:12 optimizer.py:749] 
I 03-17 23:39:12 optimizer.py:884] Considered resources (1 node):
I 03-17 23:39:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:39:12 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:39:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:39:12 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2b    0.37       [32m   âœ”[0m     
I 03-17 23:39:12 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:39:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:39:12 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2b).
[2mW 03-17 23:45:40 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2a, us-west-2c, us-west-2d.[0m
W 03-17 23:45:40 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:45:40 cloud_vm_ray_backend.py:2097] 
W 03-17 23:45:40 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:45:41 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:45:41 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:45:41 optimizer.py:749] 
I 03-17 23:45:41 optimizer.py:884] Considered resources (1 node):
I 03-17 23:45:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:45:41 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:45:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:45:41 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1b   0.37       [32m   âœ”[0m     
I 03-17 23:45:41 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:45:41 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:45:41 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1b).
[2mW 03-17 23:45:59 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 23:46:00 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:46:00 cloud_vm_ray_backend.py:2097] 
W 03-17 23:46:00 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:46:00 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:46:00 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:46:00 optimizer.py:749] 
I 03-17 23:46:00 optimizer.py:884] Considered resources (1 node):
I 03-17 23:46:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:46:00 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 23:46:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 23:46:00 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2a   0.38       [32m   âœ”[0m     
I 03-17 23:46:00 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 23:46:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:46:00 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a).
[2mW 03-17 23:46:11 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 23:46:13 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:46:13 cloud_vm_ray_backend.py:2097] 
W 03-17 23:46:13 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:46:13 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:46:13 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:46:13 optimizer.py:749] 
I 03-17 23:46:13 optimizer.py:884] Considered resources (1 node):
I 03-17 23:46:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:13 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:46:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:13 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3b    0.38       [32m   âœ”[0m     
I 03-17 23:46:13 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:46:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:46:13 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b).
[2mW 03-17 23:46:20 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3c.[0m
W 03-17 23:46:21 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:46:21 cloud_vm_ray_backend.py:2097] 
W 03-17 23:46:21 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:46:21 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:46:21 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:46:21 optimizer.py:749] 
I 03-17 23:46:21 optimizer.py:884] Considered resources (1 node):
I 03-17 23:46:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:21 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:46:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:21 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1a    0.39       [32m   âœ”[0m     
I 03-17 23:46:21 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:46:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:46:21 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a).
[2mW 03-17 23:46:28 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 23:46:28 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 23:46:28 cloud_vm_ray_backend.py:2097] 
W 03-17 23:46:28 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 23:46:28 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 23:46:28 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 23:46:28 optimizer.py:749] 
I 03-17 23:46:28 optimizer.py:884] Considered resources (1 node):
I 03-17 23:46:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:28 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 23:46:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 23:46:28 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2a    0.40       [32m   âœ”[0m     
I 03-17 23:46:28 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 23:46:28 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-23-35-56-601204/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 23:46:28 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a).
